### PR TITLE
Add more typedoc comments to layer 1 classes, types, and interfaces

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add more typedoc comments to layer 1 classes, types, and interfaces
- Remove some deprecated source files
- Comment out the typedoc comments of some hidden source code
- Rename EventChannelConnectionOptions

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1207

### CODE REVIEW

**Review by jiyoung-an — APPROVED** [review]
LGTM, Thanks!

File: src/ain.ts [hunk 1]
```diff
@@ -108,72 +125,65 @@ export default class Ain {
...
    * @return {Promise<any>}
    */
   async sendSignedTransaction(signature: string, txBody: TransactionBody, isDryrun: boolean = false): Promise<any> {
     return this.signer.sendSignedTransaction(signature, txBody, isDryrun);
   }
 
   /**
-   * Sends signed transactions to the network.
-   * @param {TransactionInput[]} transactionObjects
+   * Sends signed multiple transactions in a batch to the network.
```
Comment (woomurf) [inline src/ain.ts:176]: How about "Signs and Sends multiple transactions ..." like `sendTransaction`? I think that user does not input signed transaction.

**Review by shyun-comcom — APPROVED** [review]
LGTM!

**Review by woomurf — APPROVED** [review]
LGTM with minor question. Thanks!

File: src/ain.ts (hunk 1 above)
Comment (platfowner) [reply src/ain.ts]: Done, thanks!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an @shyun-comcom @woomurf  Now merging..

**Pull request merged**